arcade/draw_commands.py

```diff
@@ -58,10 +57,49 @@
     "draw_scaled_texture_rectangle",
     "draw_texture_rectangle",
     "draw_lbwh_rectangle_textured",
+    "get_points_for_thick_line",
     "get_pixel",
     "get_image"
 ]
 
+
+def get_points_for_thick_line(start_x: float, start_y: float,
+                              end_x: float, end_y: float,
+                              line_width: float) -> Tuple[Point2, Point2,
+                                                          Point2, Point2]:
+    """
+    Function used internally for Arcade. OpenGL draws triangles only, so a thick
+    line must be two triangles that make up a rectangle. This calculates and returns
+    those points.
+    """
+    vector_x = start_x - end_x
+    vector_y = start_y - end_y
+    perpendicular_x = vector_y
+    perpendicular_y = -vector_x
+    length = math.sqrt(vector_x * vector_x + vector_y * vector_y)
+    if length == 0:
+        normal_x = 1.0
+        normal_y = 1.0
+    else:
+        normal_x = perpendicular_x / length
+        normal_y = perpendicular_y / length
+
+    half_width = line_width / 2
+    shift_x = normal_x * half_width
+    shift_y = normal_y * half_width
+
+    r1_x = start_x + shift_x
+    r1_y = start_y + shift_y
+    r2_x = start_x - shift_x
+    r2_y = start_y - shift_y
+    r3_x = end_x + shift_x
+    r3_y = end_y + shift_y
+    r4_x = end_x - shift_x
+    r4_y = end_y - shift_y
+
+    return (r1_x, r1_y), (r2_x, r2_y), (r4_x, r4_y), (r3_x, r3_y)
```

Comment:
I know we're trying to move to returning vec2's is this something we want in this function too?

Reply:
Probably. Same as the previous comment, but it also depends on whether we make this function internal or not. The docstring currently marks it as such, but it's exposed and named as if it isn't.